build/ci/github_actions/driver/wait_for_k8s_ready.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -xe
+set +o pipefail
+
+is_pods_ready (){
```

Comment:
```suggestion
are_pods_ready (){
```

Reply:
@zingero please resolve conversations you opened before approving the PR (unless they are open for a future ticket or something)